teachertool/src/state/actions.ts

```diff
@@ -55,6 +55,11 @@ type SetRubric = ActionBase & {
     rubric: Rubric;
 };
 
+type SetConfirmationProps = ActionBase & {
```

Comment:
Can we change this name (and various related identifiers) to match?
```suggestion
type SetConfirmationOptions = ActionBase & {
```